storage-node/src/services/webApi/controllers/filesApi.ts

```diff
@@ -120,20 +119,14 @@ export async function uploadFile(
     await fsPromises.rename(fileObj.path, newPath)
     cleanupFileName = newPath
 
-    if (!accepted) {
-      await acceptPendingDataObjects(
-        api,
-        bagId,
-        res.locals.storageProviderAccount,
-        workerId,
-        uploadRequest.storageBucketId,
-        [uploadRequest.dataObjectId]
-      )
-    } else {
-      logger.warn(
-        `Received already accepted data object. DataObjectId = ${uploadRequest.dataObjectId} WorkerId = ${workerId}`
-      )
-    }
+    await acceptPendingDataObjects(
+      api,
+      bagId,
+      res.locals.storageProviderAccount,
+      workerId,
+      new BN(uploadRequest.storageBucketId),
+      [new BN(uploadRequest.dataObjectId)]
```

Comment:
It's not a huge issue, but when doing validation you're using:
```
  const storageBucketId = Number(req.query.storageBucketId)
  const dataObjectId = Number(req.query.dataObjectId)
```
But here you're using `new BN(...)`. There will be some differences in how strings are converted to `Number` vs to `BN`, so ideally you'd use the same method for both the validation & here inside `uploadFile` function.

Reply:
I have changed types in validation code to `BN` too, since the Number conversion can overflow as JS numbers precision is 53 bits while `storageBucketId` & `dataObjectId` are defined as `u64` in runtime